README.md

```diff
@@ -30,17 +30,25 @@ import { BaseTrie as Trie } from 'merkle-patricia-tree'
 const db = level('./testdb')
 const trie = new Trie(db)
 
-await trie.put(Buffer.from('test'), Buffer.from('one'))
-const value = await trie.get(Buffer.from('test'))
-console.log(value.toString())
+async function test() {
+  await trie.put(Buffer.from('test'), Buffer.from('one'))
+  const value = await trie.get(Buffer.from('test'))
+  console.log(value.toString()) // 'one'
+}
+
+await test()
```

Comment:
Removing the await since it won't execute unless it's in an async function itself.
```suggestion
test()
```

Reply:
right thanks